### PR TITLE
Fix a bug that returns class string incorrectly

### PR DESCRIPTION
**Pull request by LiberiFatali**
ouput predicted index must be subtracted by 1

### CODE REVIEW

File: tensorflow_serving/example/inception_export.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def export():
     # the human readable class description for the i'th index.
     class_tensor = tf.constant([texts[s] for s in synsets])
 
-    classes = tf.contrib.lookup.index_to_string(tf.to_int64(indices),
+    classes = tf.contrib.lookup.index_to_string(tf.sub(tf.to_int64(indices), tf.constant(1, dtype=tf.int64)),
```
Comment (kirilg) [inline tensorflow_serving/example/inception_export.py]: nit: please format it so each line is <80 chars.
(and please squash commits when you're done)

**Comment by fangweili** [comment]
Looks good. Thanks a lot for the PR.

**Comment by kirilg** [comment]
Thanks @LiberiFatali. Before we can merge, please squash all commits into one.

**Comment by kirilg** [comment]
Actually never mind, I can squash on our side. Merged.

**Pull request merged**